pkg/auth/handler/webapp/authflow_controller.go

```diff
@@ -673,6 +673,16 @@ func (c *AuthflowController) DelayScreen(r *http.Request,
 	return screen, nil
 }
 
+func (c *AuthflowController) AdvanceDirectly(
+	p string,
```

Comment:
Because `string` type cannot represent what `p` means, we should give a meaningful variable name to `p`